__tests__/interceptors/checkReRun.test.js

```diff
@@ -3,34 +3,58 @@ const CheckReRun = require('../../lib/interceptors/checkReRun')
 const Helper = require('../../__fixtures__/helper')
 require('object-dot').extend()
 
-test('#possibleInjection', () => {
-  let checkReRun = new CheckReRun()
-
-  expect(
-    checkReRun.possibleInjection(Helper.mockContext(), {id: 1}, {id: 1})
-  ).toBe(false)
-  expect(
-    checkReRun.possibleInjection(Helper.mockContext(), {id: 1}, {id: 2})
-  ).toBe(true)
-})
-
-test('#process', async () => {
+test('return if pre conditions are not met', async () => {
   let checkReRun = new CheckReRun()
   let context = Helper.mockContext()
 
   context.event = 'check_run'
-  context.payload.action = 'rerequested'
+  context.payload.action = 'opened'
   Object.set(context, 'payload.check_run.output.text', mockOutput())
-  context.payload.check_run.pull_requests = [{number: 1}]
-  context.payload.check_run.id = 123
   context.github.pullRequests.get.mockReturnValue({ data: { number: 456 } })
   let newContext = await checkReRun.process(context)
 
-  expect(newContext.payload.pull_request.number).toBe(456)
+  expect(newContext.event).toBe('check_run')
+
+  context.payload.action = 'rerequested'
+
+  newContext = await checkReRun.process(context)
+  expect(newContext.event).toBe('check_run')
+
+  context.payload.check_run.pull_requests = [{number: 1}]
+  context.payload.check_run.id = 123
+
+  newContext = await checkReRun.process(context)
   expect(newContext.event).toBe('pull_request')
-  expect(newContext.payload.action).toBe('unlabeled')
 })
 
+// test('#possibleInjection', () => {
```

Comment:
May I ask the reasons for this entire block to be commented out?

Reply:
forgot to uncomment